frontend/app_flowy/lib/startup/tasks/app_widget.dart

```diff
@@ -82,8 +83,10 @@ class ApplicationWidget extends StatelessWidget {
         builder: (context, state) => MaterialApp(
           builder: overlayManagerBuilder(),
           debugShowCheckedModeBanner: false,
-          theme: state.theme.getThemeData(state.locale),
-          darkTheme: state.darkTheme.getThemeData(state.locale),
+          theme: getThemeData(
```

Comment:
Would it be better if call `state.darkTheme `or `state.lightTheme` directly? Just by moving these constructions to the Cubit.

Reply:
I've added a commit to address this